src/main/java/com/blackducksoftware/integration/hub/alert/StartupManager.java

```diff
@@ -82,7 +83,12 @@ public void logConfiguration() {
         logger.info("Hub Proxy Host:     {}", globalProperties.getHubProxyHost());
         logger.info("Hub Proxy Port:     {}", globalProperties.getHubProxyPort());
         logger.info("Hub Proxy User:     {}", globalProperties.getHubProxyUsername());
-        logger.info("Hub Proxy Password: **********");
+        final String proxyPasswordLabel = "Hub Proxy Password:";
```

Comment:
Should we apply this same logic to Hub API Key?

Reply:
I think this was only done because the string was repeated.

Reply:
Yeah I would have repeated the string so I just defined it once